README.md

```diff
@@ -289,3 +289,43 @@ restore the db file
 ```bash
 docker exec -it <id> psql -U postgres -d claim -f /tmp/dump_file
 ```
+
+Alternate instructions
+
+Run
+
+`docker run -d -p 5432:5432 -e POSTGRES_PASSWORD=postgres --name postgres-db postgres`
+
+ensure you have a .env file
+
+```
+PORT=9000
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/claim"
+ACCESS_SECRET=dPEBknfdAcx5bir34KnX2mATWZnvM4xF
```

Comment:
hey @Emos21 blank out the secrets in the pr, don't check in actual secrets - assume these are already compromised now so you should rotate them

Reply:
aside from that this is great